lib/cli/src/commands/create_exe.rs

```diff
@@ -1357,30 +1362,36 @@ impl LinkCode {
 #[cfg(feature = "http")]
 mod http_fetch {
     use anyhow::{anyhow, Context, Result};
-    use http_req::{
-        request::Request,
-        response::{Response, StatusCode},
-        uri::Uri,
-    };
+    use http_req::{request::Request, response::StatusCode, uri::Uri};
     use std::convert::TryFrom;
+    use target_lexicon::OperatingSystem;
 
     pub fn get_latest_release() -> Result<serde_json::Value> {
         let mut writer = Vec::new();
         let uri = Uri::try_from("https://api.github.com/repos/wasmerio/wasmer/releases").unwrap();
 
-        let response = Request::new(&uri)
-            .header("User-Agent", "wasmer")
+        // Increases rate-limiting in GitHub CI
+        let auth = std::env::var("GITHUB_TOKEN");
+        let mut response = Request::new(&uri);
+
+        if let Ok(token) = auth {
+            response.header("Authorization", &format!("Bearer {token}"));
+        }
+
+        let response = response
+            .header("User-Agent", "wasmerio")
```

Comment:
I'm not sure if people check user agents much any more, but we can use `env!("CARGO_PKG_XXX")` variables to include more information if we want. 

For example, 

```rust
const USER_AGENT: &str = concat!(
  env!("CARGO_PKG_NAME"),
  " v"
  env!("CARGO_PKG_VERSION"),
);

```

Reply:
According to GitHub, the User-Agent should be the organization name, and there is no way to get that from env vars.

Reply:
Ah thank, I didn't know that.